### PR TITLE
E4s 21.08 arcticus

### PR DESCRIPTION
**Pull request by frankwillmore**
Localization for JLSE. Notable changes include:

- Using only a single spack clone for all of the individual build stages. This has resulted in a more elegant build pipeline, greatly reducing load on I/O resources, and thereby eliminating many intermittent pipeline failures. 
- OneAPI builds of:
  - hdf5@1.12.0  
  - papi@6.0.0.1 
  -  parallel-netcdf@1.12.2 
  -  parsec@3.0.2012
- Addition of new test stage to CI pipeline which runs the E4S testsuite

### CODE REVIEW

**Comment by frankwillmore** [comment]
@eugeneswalker @shahzebsiddiqui 
Would you please review when you get a chance?

**Review by eugeneswalker — APPROVED** [review]
Thank you!

File: ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt [hunk 1]
```diff
@@ -0,0 +1,208 @@
+==> In environment /soft/ecp/spack-configs/ANL/JLSE/Arcticus/E4S-21.08/prod
```
Comment (shahzebsiddiqui) [inline ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt:1]: can you please run the following command `spack find --format "{name},{version},{compiler},{variants},{arch}"` when generating the file to be consistent with other stacks. see https://github.com/spack/spack-configs/blob/main/NERSC/cori/e4s-21.05/e4s-21.05.txt#L1.

Its up to you if you want to preserve the command as the first line or not. I think @wspear will be looking at parsing this content though i am not sure if this standard for generating spack find output works for him.

Comment (frankwillmore) [reply ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt:1]: @wspear is this format what you need, or is there something else you would prefer?

Comment (wspear) [reply ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt:1]: Yep. I'm already using https://raw.githubusercontent.com/spack/spack-configs/main/ANL/JLSE/Arcticus/E4S-21.05/e4s-21.05.txt so continuing with that output should be fine unless we decide we need more/different info about the deployment.

Comment (frankwillmore) [reply ANL/JLSE/Arcticus/E4S-21.08/e4s-21.08.txt]: @wspear Thanks for confirming. @shahzebsiddiqui I have reformatted the output and added it to the PR.

**Comment by shahzebsiddiqui** [comment]
thanks @frankwillmore this looks good. Ill merge this. When you have chance can you update this table in E4S facility dashboard https://e4s.readthedocs.io/en/latest/facility_e4s.html for Articus E4S-21.08.

**Pull request merged**